jenkins/Dockerfile-blossom.ubuntu

```diff
@@ -25,7 +25,11 @@
 
 ARG CUDA_VER=11.0
 ARG UBUNTU_VER=18.04
+ARG UCX_VER=v1.11.0
 FROM nvidia/cuda:${CUDA_VER}-runtime-ubuntu${UBUNTU_VER}
+ARG CUDA_VER
```

Comment:
dup with line #26?

Reply:
no this is required so we can use the ARG after the FROM.

Reply:
@zhanga5 as Alessandro said, this is a must have. 
We put arg before FROM to get the image name, and need to re-declare to make sure we can still access it after FROM since they are different layers in docker images~